src/opengeodeweb_back/geode_functions.py

```diff
@@ -13,43 +13,47 @@
 import werkzeug
 
 # Local application imports
-from .geode_objects import objects_list
+from .geode_objects import geode_objects_dict
```

Comment:
Why this change? This implies breaking change which implies. Be careful about not adding too much work

Reply:
The name was a little bit confusing, it said list but it was actually a dict.
And since the name is just used in the functions, this isn't supposed to introduce any breaking change